translations/default/AdminDesignHelp.xlf

```diff
@@ -218,7 +218,8 @@
       <trans-unit id="cabfe7563d12c3e047000dc5703d9709">
         <source>Add new page category</source>
         <target>Add new page category</target>
-        <note>Line: 31</note>
+        <note>Line: 31
+Comment: Stable feature flags form</note>
```

Comment:
The comment feels weird here 🤔

Reply:
That's a feature from TranslationToolBundle, it picks any comment around the line where the wording is picked up and adds it. This particular comment is weird though.